### ISSUE TITLE
missing dependency to python version in pip package

### ISSUE THREAD

**Issue by freedge** [netaddr/netaddr#365]
```
podman run -ti --rm --entrypoint=bash python:3.6
pip install netaddr
python
import netaddr
```

getting
```
RuntimeError: Python 3.7.0 or higher is required!
```

expecting instead that pip should download "netaddr<1.0" that is compatible with python 3.6

Is there some way to add a dependency to  python_version >= 3.7 so that pip just resolves to a working version?

Thanks!

**Comment by jstasiak** [comment]
Hey @freedge, sorry about that. Yeah there is, I'll add it: https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-requires-python

**Comment by jstasiak** [comment]
Addressed in https://github.com/netaddr/netaddr/commit/eae99e0ee86c49de7d2fd44a65e3f96002c5b9a9, not sure when I release that (when I do I think I'll yank 1.0.0 from PyPI).

**Comment by andy-maier** [comment]
Thanks for the fix! (I was just about to open an issue and provide a PR for that and found this issue here :-)

**Comment by jstasiak** [comment]
Fixed in 1.1.0 released earlier today, I yanked 1.0.0 from PyPI so this shouldn't cause more trouble for anyone else in the future.
